### PR TITLE
meppp:0.2.0

### PR DESCRIPTION
**Pull request by sjfhsjfh**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: update lib structure & README

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by laurmaedje** [comment]
The `meppp` package was submitted by @CL4R3T, so merging this requires their approval.

**Comment by CL4R3T** [comment]
I agree this merge

**Comment by CL4R3T** [comment]
> The `meppp` package was submitted by @CL4R3T, so merging this requires their approval.

I agree.

File: packages/preview/meppp/0.2.0/template/main.pdf
Comment (laurmaedje) [inline packages/preview/meppp/0.2.0/template/main.pdf:1]: This file should be deleted.

Comment (sjfhsjfh) [reply packages/preview/meppp/0.2.0/template/main.pdf:1]: Deleted

**Pull request merged**